src/ttmask/sphere.py

```diff
@@ -1,6 +1,37 @@
 from ._cli import cli
+import numpy as np
+import einops
+import napari
+import typer
 
+typer.main.get_command_name = lambda name: name
```

Comment:
this makes sense, do you understand what it's doing? :) can explain if not

as mentioned: would recommend removing and just having life be simple and dictated by the opinionated tool :)